src/agenda/index.js

```diff
@@ -44,6 +44,8 @@ export default class AgendaView extends Component {
     onDayPress: PropTypes.func,
     /** callback that gets called when day changes while scrolling agenda list */
     onDaychange: PropTypes.func,
+    /** callback that gets called on header press */
+    onHeaderPress: PropTypes.func,
```

Comment:
What about `onHeaderLongPress`? Where are you passing this prop to the CalendarList component?